src/omv/modules/py_image.h

```diff
@@ -10,7 +10,7 @@
  */
 #ifndef __PY_IMAGE_H__
 #define __PY_IMAGE_H__
-#include "imlib.h"
+#include "py_helper.h"
```

Comment:
```suggestion
```
Please include this directly in the C file(s) where py_helper.h is needed.

Reply:
Fixed